macaroonbakery/tests/test_authorizer.py

```diff
@@ -2,10 +2,8 @@
 # Licensed under the LGPLv3, see LICENCE file for details.
 from unittest import TestCase
 
-from macaroonbakery.identity import Identity, SimpleIdentity, ACLIdentity
+import macaroonbakery
 from macaroonbakery import checkers
```

Comment:
import macaroonbakery.checkers as checkers

(and any others I've omitted elsewhere)